src/jenesis/cmd/add/deployment.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import os
+import toml
+
+from jenesis.config import Config
+
+
+def run(args: argparse.Namespace):
+
+    project_root = os.path.abspath(os.getcwd())
+    contract_root = os.path.join(project_root, "contracts", args.contract)
+
+    # check that we are actually running the command from the project root
+    if not os.path.exists(os.path.join(project_root, "jenesis.toml")):
+        # pylint: disable=all
```

Comment:
What exactly needs to be disabled here? Duplicated code?

Reply:
Thought it was giving duplicated code issue, but just removed it and all good